### ISSUE TITLE
quay.io/podman tags don't match included podman version

### ISSUE THREAD

**Issue by paleozogt** [containers/libpod#5357]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

The podman images on `quay.io` have tags that don't match the version that's in the image.

**Steps to reproduce the issue:**

Tag v1.6.2 has v1.6.2 in it:

```
➜  docker run --rm quay.io/podman/stable:v1.6.2 podman version
Version:            1.6.2
RemoteAPI Version:  1
Go Version:         go1.12.10
OS/Arch:            linux/amd64
```

But tag v1.6.5 has v1.6.2 in it (O_o):
```
➜  docker run --rm quay.io/podman/stable:v1.6.5 podman version
Version:            1.6.2
RemoteAPI Version:  1
Go Version:         go1.13.1
OS/Arch:            linux/amd64
```

Similarly for v1.8.1-rc2:

```
➜  docker run --rm quay.io/podman/stable:v1.8.1-rc2 podman version
Version:            1.6.2
RemoteAPI Version:  1
Go Version:         go1.13.1
OS/Arch:            linux/amd64
```

**Describe the results you received:**

The podman versions don't match the image tag.

**Describe the results you expected:**

The version of podman in an image should match the tag of the image.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

See above.

**Output of `podman info --debug`:**

n/a

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

n/a

**Additional environment details (AWS, VirtualBox, physical, etc.):**

n/a

**Comment by mheon** [comment]
@TomSweeneyRedHat I think you maintain this image?

**Comment by TomSweeneyRedHat** [comment]
Yes.  Unfortunately when we tag a new Podman release quay.io automatically builds a "stable" version with that release number as a tag.  However, the new release hasn't been made into an rpm yet, never mind getting it through the stable release platform.  So I need to remove them "by hand" and then create the image once it reaches stable.  

I'll clean that up in the next day or two.

**Comment by TomSweeneyRedHat** [comment]
FWIW, I've removed the badly labled tags from that repo.  Podman v1.8 is not yet in stable and our first test sadly pointed to a packaging problem that we'll need to tend to.  I also tried to build a v1.7 variant, but found issues with that RPM.  Will post later versions once they become stable.

**Comment by paleozogt** [comment]
It looks like 1.8.0 is available via `yum`, but not the other post 1.6.2 builds:
```
[root@c9946fba3dbc /]# yum --showduplicates list podman
Last metadata expiration check: 0:04:11 ago on Tue Mar  3 16:40:51 2020.
Installed Packages
podman.x86_64                                                 2:1.6.2-2.fc31                                                 @fedora
Available Packages
podman.x86_64                                                 2:1.6.2-2.fc31                                                 fedora
podman.x86_64                                                 2:1.8.0-2.fc31                                                 updates
```

**Comment by rhatdan** [comment]
Should be availabel now.
